config.py

```diff
@@ -6,6 +6,7 @@
 PHRASES = [
     'пернул мозгом',
     'да, ништяк',
+    'ебать в очки коней',
```

Comment:
@ht-albert tnx for pr, please rewrite to `aiotg` 😆

Reply:
please, read the added phrase, this is the answer to your comment 😡 😤